lib/utils.libsonnet

```diff
@@ -56,4 +56,8 @@
           metric: labels.metric,
         },
       },
+
+  // if clusterLabel is set, return the string, otherwise return an empty string
+  ifClusterLabelSet(config, string)::
+    if std.length(config.clusterLabel) > 0 then string else '',
```

Comment:
The `config.clusterLabel` is set to `cluster` by default, so you would intentionally have to clear that label in this case.

Can you use `config.showMultiCluster` boolean instead for this conditional? That is default `false` and feels more like an explicit toggle on/off. Then the util function might be better named `ifShowMultiCluster`.

Reply:
Done 👍